datasets/fr/maires/crawler.py

```diff
@@ -0,0 +1,85 @@
+"""
+Crawler for active French mayors.
+"""
+
+import csv
+from typing import Dict
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+UNUSED_FIELDS = [
+    "Code sexe",
```

Comment:
sex or gender is also nice to have, acknowledging that we're probably not treating it with as much nuance as we should, but the aim is just to reflect the source

Reply:
Ah, of course! Yes also useful for disambiguating, I just didn't bother to look in the data dictionary to see what the right attribute name was.